src/app/container/launch/launch.component.html

```diff
@@ -31,7 +31,7 @@
         <div class="col-sm-4">
           <mat-form-field class="m-2">
             <mat-select name="toolVersions" [(value)]="currentDescriptor" (selectionChange)="reactToDescriptor($event)">
-              <mat-option *ngFor="let descriptor of descriptors" [value]="descriptor">{{descriptor}}</mat-option>
+              <mat-option *ngFor="let descriptor of validDescriptors" [value]="descriptor">{{descriptor}}</mat-option>
```

Comment:
The name `validDescriptors` is very misleading considering it doesn't have anything to do with validation.  Change it in the .ts file.